packages/armbian/builddeb

```diff
@@ -289,15 +289,17 @@ EOF
 	chmod 755 "$tmpdir/DEBIAN/$script"
 done
 
-##
-## Create sym link to kernel image
-##
-sed -e "s/exit 0//g" -i $tmpdir/DEBIAN/postinst
-cat >> $tmpdir/DEBIAN/postinst <<EOT
-ln -sf $(basename $installed_image_path) /boot/$image_name 2> /dev/null || mv /$installed_image_path /boot/$image_name
-touch /boot/.next
-exit 0
-EOT
+if [ "${SKIP_KERNEL_SYMLINK}" != "yes" ]; then
+	##
+	## Create sym link to kernel image
+	##
+	sed -e "s/exit 0//g" -i $tmpdir/DEBIAN/postinst
+	cat >> $tmpdir/DEBIAN/postinst <<- EOT
+	ln -sf $(basename $installed_image_path) /boot/$image_name 2> /dev/null || mv /$installed_image_path /boot/$image_name
+	touch /boot/.next
+	exit 0
+	EOT
+fi
```

Comment:
Original review by @The-going:
---------------------------------------------------------------
Development of the debian kernel package build script.
    Today we just copy two scripts to the source directory.
    This will be in the source package.
   How will this global variable be initialized when the kernel is built natively, without the armbian build system from the source package?

Reply:
Hmm. I had _not_ considered this would be built outside the build system, but is indeed valid question.
Anyway, the same question could be asked of ALL the variables passed in https://github.com/armbian/build/blob/1831d83acc9ef651f4f4d38eaccdbc8d2357c2b0/lib/compilation.sh#L513-L530

I would much prefer if we used the standard builddeb from upstream kernel, since that does not do that symlink.

Reply:
Reply from @The-going:
----------------------------------------------------------------------------------------------------------------

Good health, Ricardo.
> Anyway, the same question could be asked of ALL the variables passed in

That's right, you're right. It is for this reason that the discussion has a general character.

> I would much prefer if we used the standard builddeb from upstream kernel, since that does not do that symlink.

[Standard scripts](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/scripts/package) from the kernel sources are very good and working examples\templates,
but this is not enough for industrial release.
Armbian has improved these scripts so that they do a little more.
My question is not how you handled the exception today and removed the creation of a symbolic link, but how we plan to do it in the future. I want to hear your opinion as an author.
I hope I wasn't too intrusive.

Reply:
Not intrusive at all! Very much appreciated. @The-going 
I think it might be interesting to handle the packaging maybe as an extension itslef, so different packaging needs might have different solutions. For example, for UEFI, we could maybe just use the standard packaging!  I will propose solution in a separate commit for review, in this same PR/branch.

Reply:
We should try to avoid using standard packaging altogether.
This is my inner conviction.

Reply:
Addressed by removing the hack, and fixing the missing `$image_name` that was the original problem.
Thanks @The-going  for driving this investigation and de-hacking !